Assets/Scripts/Gameplay/Action/ActionID.cs

```diff
@@ -7,15 +7,10 @@ namespace Unity.BossRoom.Gameplay.Actions
     /// This struct is used by Action system (and GameDataSource) to refer to a specific action in runtime.
```

Comment:
LobbyPlayerState can't be converted as well? Does the name string property mess with it?
The player name inside, no, but the containing state?

Reply:
Converting LobbyPlayerState over to implement `INetworkSerializeByMemcpy`  does work, but it'd introduce bandwidth overhead that I don't think is worth having performant serailzation. Per offline discussion:

> We try to encourage the use of FixedString for all network purposes because of this (but putting FixedString inside an INetworkSerializeByMemcpy struct is not recommended because it will lose the bandwidth optimization - an empty FixedString128Bytes serialized normally or through INetworkSerializable will use 4 bytes of bandwidth, but inside an INetworkSerializeByMemcpy, that same empty value would consume 132 bytes of bandwidth

Reply:
The fixed string is wrapped by an INetworkSerializable though, do we lose that optim if that wrapper is then inside a INetworkSerializeByMemcpy ?

Reply:
ah right, it copies the whole struct, byte by byte and won't call the serialize method in the child struct. nvm then

Reply:
it could be worth adding a quick comment in our LobbyPlayerState struct?